htdocs/composer.json

```diff
@@ -0,0 +1,85 @@
+{
+	"name": "impresscms/impresscms",
+	"description": "ImpressCMS - A dynamic and user-friendly Content Management System",
+	"type": "project",
+	"keywords": ["cms", "content-management", "php", "mysql", "web"],
+	"homepage": "https://www.impresscms.org/",
+	"license": "GPL-2.0-or-later",
+	"authors": [
+		{
+			"name": "The ImpressCMS Project",
+			"homepage": "https://www.impresscms.org/",
+			"role": "Developer"
+		}
+	],
+	"require": {
+		"php": ">=7.4.0",
+		"ext-gd": "*",
+		"ext-json": "*",
+		"ext-mbstring": "*",
+		"ext-mysqli": "*",
+		"ext-pcre": "*",
+		"ext-pdo": "*",
+		"ext-session": "*",
+		"ext-xml": "*",
+		"ext-zlib": "*",
+		"composer/composer": "2.8.12",
+		"ezyang/htmlpurifier": "^4.19",
+		"simplepie/simplepie": "^1.9",
+		"smottt/wideimage": "^1.1"
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^9.0",
+		"squizlabs/php_codesniffer": "^3.0"
```

Comment:
imho both these should be added as separate PR's with config files that doesn't exists here